RunningOrder/Story/Story.swift

```diff
@@ -12,6 +12,7 @@ struct Story {
     let name: String
     let ticketReference: String
     let epic: String
+    var environment: String
```

Comment:
Why did you add this var to the model ?

Reply:
To check something to modify and ensure everything works well :)

Reply:
You can remove it as it is not part of real model